swift/trainers/kto_trainer.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, *args, template: Template, test_oom_error=False, **kwargs):
                 fn_kwargs={'template': template},
                 desc='Encode dataset with template',
             )
+        is_multimodal = kwargs.pop('is_multimodal')
```

Comment:
KTO currently does not support vision models; The modification here is just to prevent conflicts